x/migration/module/cmd/collect_morse_accounts_test.go

```diff
@@ -16,7 +16,7 @@ import (
 	migrationtypes "github.com/pokt-network/poktroll/x/migration/types"
 )
 
-// TODO_MAINNET(@bryanchriswhite): Add an E2E/integration test using real data.
+// TODO_MAINNET_MIGRATION(@bryanchriswhite): Add an E2E/integration test using real data.
```

Comment:
This is the migration snapshot feature which is now merged and has its own TODOs.
```suggestion
```